### ISSUE TITLE
Neo4j connection (Read only Mode)

### ISSUE THREAD

**Issue by alwayscheers** [neo4j-labs/llm-graph-builder#927]
I deployed the community edition of Neo4j 5.21.0 on a Linux server, and when connecting to the LLM Graph Builder, I encountered a "Neo4j connection (Read only Mode)" issue. What could be the possible reasons? Does the LLM Graph Builder web service only accept the enterprise version?
![image](https://github.com/user-attachments/assets/3d3413ee-88c0-41fc-aed2-546ce74da364)
![image](https://github.com/user-attachments/assets/f0fb8922-2086-4e0a-ba66-febe65d4f494)

**Comment by kartikpersistent** [comment]
Hi @alwayscheers  you can refer this issue 
#839

**Comment by kartikpersistent** [comment]
yes we are planning to do a new release this week with latest changes

**Comment by kartikpersistent** [comment]
Hi @neverlatetolearn0  and @alwayscheers  we released latest version previous weak only so please test and let us if you are still facing the same issue?

**Comment by neverlatetolearn0** [comment]
> Hi @neverlatetolearn0 and @alwayscheers we released latest version previous weak only so please test and let us if you are still facing the same issue?

Yes, but I've now modified the check_account_access function in the graphDB_dataAccess.py file to return True when an exception is thrown。 I found the solution from #839
